docs/edgeql/funcops/datetime.rst

```diff
@@ -505,6 +503,8 @@ Date and Time
     value can be parsed from the input :eql:type:`str` *s* with an
     optional *fmt* argument or it can be given in terms of its
     component parts: *year*, *month*, *day*, *hour*, *min*, *sec*.
+    For more details on formatting see :ref:`here
```

Comment:
Please make "For more details" a separate paragraph in all places

Reply:
OK